recipes/pytest-pyvista/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "pytest-pyvista" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pytest_pyvista/pytest_pyvista-{{ version }}.tar.gz
+  sha256: 43e6f0a85b6ff23aff51b7f750528de0fdda6da2011a94b636f05d8498fe38a0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - python >=3.8
+    - pytest >=3.5.0
+    - pyvista
+
```

See https://github.com/pyvista/pytest-pyvista/blob/597da3865bb9f548fb66242ef37ed72cfa4a4af7/pyproject.toml#L2

```suggestion
    - flit_core >=3.2,<4

```